qwik-graphql-tailwind/src/components/user-dropdown/user-dropdown.tsx

```diff
@@ -24,8 +25,17 @@ export const UserDropdown = component$(({ image, username }: UserDropdownProps)
     store.expanded = false;
   });
 
-  const signOut$ = $(() => {
-    // TODO: sign out
+  const signOut$ = $(async () => {
+    sessionStorage.removeItem(AUTH_TOKEN);
+    fetch(SIGN_OUT_URL, {
+      method: 'POST',
+    })
+      .then(() => {
+        window.location.href = '/auth/signin';
```

Comment:
is this the only way qwik offers for now?

Reply:
This is a feature of Qwik, [here](https://qwik.builder.io/docs/advanced/optimizer/#the-meaning-of-)

Reply:
are you talking about the fetch function and the redirection?

Reply:
The only thing qwik here is the $(), and I believe you now know what it is for.